src/test/java/Homework18.java

```diff
@@ -6,23 +6,28 @@
 public class Homework18 extends BaseTest {
 
     @Test
-    public void playASongTest() {
+    public static void playASongTest() throws InterruptedException {
+        launchBrowser(url);
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

```
  @Test
    public static void playASongTest() throws InterruptedException {
        login("demo@class.com", "te$t$tudent");
        clickSubmit();

        Thread.sleep(2000);

//        //click 'play' button
        WebElement pressButton = driver.findElement(By.cssSelector("[data-testid='play-next-btn']"));
        pressButton.click();

        WebElement pressPlayButton = driver.findElement(By.cssSelector("[data-testid='play-btn']"));
        pressPlayButton.click();

        Thread.sleep(2000);

        WebElement validateSongIsPlaying = driver.findElement(By.cssSelector("div>img"));
        Assert.assertTrue(validateSongIsPlaying.isDisplayed());

        Thread.sleep(2000);

        closeBrowser();
    }

}
```

![image](https://user-images.githubusercontent.com/104761482/211746937-a3ca54e0-e419-41e5-8ddb-8c308a4a0089.png)